### ISSUE TITLE
Firewall: policies are not showed if no rules are present

### ISSUE THREAD

**Issue by federicoballarini** [NethServer/dev#6432]
**Steps to reproduce**

- Login into Cockpit
- Go under Firewall > Rules
- The page must be blank, without rules.

**Expected behavior**

- Policies button must be shown

**Actual behavior**

- Policies button is hidden, up to you create a rule.

**Components**

```
nethserver-firewall-base-3.13.6-1.ns7.noarch
nethserver-firewall-base-ui-3.13.6-1.ns7.noarch
```

**See also**

[https://community.nethserver.org/t/display-active-firewall-policies/17737](https://community.nethserver.org/t/display-active-firewall-policies/17737)

----

Thanks to **carsten**

**Comment by federicoballarini** [comment]
@andre8244 can you take a look into it?

**Comment by gsanchietti** [comment]
If you do not have any rule, it's not so usefull to see the policy list.
Also default policies are well known and documented: https://docs.nethserver.org/en/v7/firewall.html#policy

So this is not for sure a bug, but a request for change.
Still, I thing the current behavior is correct.

**Comment by federicoballarini** [comment]
Hi @gsanchietti, thanks for your reply and your time :) I disagree with you, because policies are always active, also if there isn't any rule declared. Traffic from red to green is always blocked by default. I know that docs is clear, but if the button is already present I think it should be showed also if no rules are present.
It's a bug IMHO because policies are not enabled only when you write a rule, but are always enabled.
Tell me if I can help. Have a nice evening.

**Comment by gsanchietti** [comment]
If the machine has no rules, it's not acting as a real gateway but it's just a router.
In this scenario, you do not really need to see the policies because the masquerade is active and all clients can just access the internet.
Everything else is already handled automatically.

Policies are needed when you are configuring the firewall rules to know which is the default firewall behavior.

**Comment by federicoballarini** [comment]
I don't know. I leave the choice to you :)

**Comment by federicoballarini** [comment]
The only thing is. If I create a rule with no sense, like Green > Green = Accept, the button is showed. Nothing changes from no rules situation in this case, so in my opinion, show always the policies link.
Also if you have more than one NIC, policies are still active.

**Comment by gsanchietti** [comment]
If you create such a rule, it will probably not be even converted to a real iptables rule due to shorewall optimizations.
I'm still not convinced of such modification but I know that @cotosso is not against it.

If you want to propose a PR, we will try to review it, otherwise I'm going to close the issue as wontfix.

**Comment by federicoballarini** [comment]
I will try. I will update you asap, have a nice weekend :)

**Comment by nethbot** [comment]
in `7.9.2009/testing`:
- nethserver-firewall-base-3.13.6-1.2.g5126a26.ns7.noarch.rpm [x86_64](http://packages.nethserver.org/nethserver/7.9.2009/testing/x86_64/Packages/nethserver-firewall-base-3.13.6-1.2.g5126a26.ns7.noarch.rpm) [armhfp](http://packages.nethserver.org/nethserver/7.9.2009/testing/armhfp/Packages/nethserver-firewall-base-3.13.6-1.2.g5126a26.ns7.noarch.rpm) [aarch64](http://packages.nethserver.org/nethserver/7.9.2009/testing/aarch64/Packages/nethserver-firewall-base-3.13.6-1.2.g5126a26.ns7.noarch.rpm) 
- nethserver-firewall-base-ui-3.13.6-1.2.g5126a26.ns7.noarch.rpm [x86_64](http://packages.nethserver.org/nethserver/7.9.2009/testing/x86_64/Packages/nethserver-firewall-base-ui-3.13.6-1.2.g5126a26.ns7.noarch.rpm) [armhfp](http://packages.nethserver.org/nethserver/7.9.2009/testing/armhfp/Packages/nethserver-firewall-base-ui-3.13.6-1.2.g5126a26.ns7.noarch.rpm) [aarch64](http://packages.nethserver.org/nethserver/7.9.2009/testing/aarch64/Packages/nethserver-firewall-base-ui-3.13.6-1.2.g5126a26.ns7.noarch.rpm)

**Comment by gsanchietti** [comment]
**Test case 1**
- Install the package on a machine where there are no rules available
- The "Policy" link should be visible on the right

**Test case 2**
- Update a machine where the firewall already has rules
- The "Policy" link should be visible on the right

**Comment by cotosso** [comment]
Case 1 and 2 tested.
Everything works as expected

![Firewall-vmtest-vbox-dav](https://user-images.githubusercontent.com/7226896/108720543-3ea05e80-7521-11eb-96b7-8d3112b9f6fc.png)

**Comment by gsanchietti** [comment]
Verified also by **carsten**.

**Comment by nethbot** [comment]
in `7.9.2009/updates`:
- nethserver-firewall-base-3.13.7-1.ns7.noarch.rpm [x86_64](http://packages.nethserver.org/nethserver/7.9.2009/updates/x86_64/Packages/nethserver-firewall-base-3.13.7-1.ns7.noarch.rpm) [armhfp](http://packages.nethserver.org/nethserver/7.9.2009/updates/armhfp/Packages/nethserver-firewall-base-3.13.7-1.ns7.noarch.rpm) [aarch64](http://packages.nethserver.org/nethserver/7.9.2009/updates/aarch64/Packages/nethserver-firewall-base-3.13.7-1.ns7.noarch.rpm) 
- nethserver-firewall-base-ui-3.13.7-1.ns7.noarch.rpm [x86_64](http://packages.nethserver.org/nethserver/7.9.2009/updates/x86_64/Packages/nethserver-firewall-base-ui-3.13.7-1.ns7.noarch.rpm) [armhfp](http://packages.nethserver.org/nethserver/7.9.2009/updates/armhfp/Packages/nethserver-firewall-base-ui-3.13.7-1.ns7.noarch.rpm) [aarch64](http://packages.nethserver.org/nethserver/7.9.2009/updates/aarch64/Packages/nethserver-firewall-base-ui-3.13.7-1.ns7.noarch.rpm)
